.gitmodules

```diff
@@ -16,3 +16,7 @@
 [submodule "windows_docker_resources/qtaccount"]
 	path = windows_docker_resources/qtaccount
 	url = git@github.com:osrf/qtaccount
+[submodule "windows_docker_resources/ros2-cookbooks"]
+	path = windows_docker_resources/ros2-cookbooks
+	url = git@github.com:ros-infrastructure/ros2-cookbooks
+	branch = brawner/ros2-windows
```

Comment:
Does the branch parameter here need to be changed or removed before merge? I need to brush up on gitmodules config.

Reply:
Yah, this should be removed. Done